### DISCUSSION TITLE
What is PB

### DISCUSSION THREAD

**Discussion by ForesterGoGo** [Chia-Network/chia-blockchain#2138] [Q&A] [answered]
I didn't find in the FAQ what "PB"means. I take it from the words "Plant Block", right?

**Comment by hoffmang9** [comment, marked as answer]
Petabyte.

**Reply by ForesterGoGo** [reply]
Awesome, people are already starting to "stutter" about petabytes, even a couple of terabytes is not enough.

**Comment by dorkmo** [comment]
PBJ means youre jelly of other farmers' petabytes
